CONTRIBUTING.md

```diff
@@ -8,15 +8,14 @@ which explains the minimum behavior expectations for Ayo.js contributors.
 
 ## Issue Contributions
 
-When opening issues or commenting on existing issues, please make sure
-discussions are related to concrete technical issues with Ayo.js.
+When opening issues or commenting on existing issues [in this
+repository](https://github.com/ayojs/ayo/issues), discussions should be related
+to:
 
-* For general help using Ayo.js, please file an issue at the
-[ayo.js help repository](https://github.com/ayojs/ayo/issues).
-
-* Discussion of non-technical topics (such as intellectual property and
-trademark) should use the
-[Technical Steering Committee (TSC) repository](https://github.com/nodejs/TSC/issues).
+* Technical issues with Ayo.js.
+* General help using Ayo.js.
```

Comment:
I highly recommend planning to have this in it's own repo medium term like Node does.